README.md

```diff
@@ -0,0 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> First Capstone Project.
+
+
+## Built With
+
+- HTML, CSS & JavaScript
+- Bootstrap
+- Linter
+
+## Live Demo (if available)
+
+[Live Demo Link](https://mohamedck.github.io/Capstone-Project-1/)
+
+
+
+
+## Authors
+
+👤 **Mohamed Abdulhakim**
+
+- GitHub: [@githubhandle](https://github.com/MohamedCK)
+- Twitter: [@twitterhandle](https://twitter.com/MohamedCK0)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohamed-abdulhakim-2868521b6/)
+
+👤 **Praises tula**
+
+- GitHub: [@praisespjmt](https://github.com/PraisesPJMT)
+- Twitter: [@praisespjmt](https://twitter.com/PraisesPJMT)
+- LinkedIn: [Praises Musa Tula](https://www.linkedin.com/in/praises-tula-9233aa76)
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] I kindly suggest that you add appropriate credit to Cindy Shin in your Acknowledgments section, as required by the Creative Commons License


## EXAMPLE
`This template was inspired by [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)`